ServiceTier/Services/PresetService.cs

```diff
@@ -39,11 +40,11 @@ public void DeletePresetById(int presetid)
         {
```

Comment:
again - what if user doesn't own this preset. do we handle (or plan to) this scenario somehow?